ADALiOS/ADALiOS/ADAuthenticationWebViewController.m

```diff
@@ -153,12 +161,27 @@ - (BOOL)webView:(UIWebView *)webView shouldStartLoadWithRequest:(NSURLRequest *)
         // This event is explicitly scheduled on the main thread as it is UI related.
         NSAssert( nil != _delegate, @"Delegate object was lost" );
         
-        dispatch_async( dispatch_get_main_queue(), ^{ [_delegate webAuthenticationDidCompleteWithURL:request.URL]; } );
```

Comment:
Going to move this into the mentioned block, the resulting code will be clearer. :+1: